lib/stack/trackNewConversation.tsx

```diff
@@ -0,0 +1,28 @@
+import { Address } from "viem";
+import getStackClient from "./getStackClient";
+import {
+  CHAT_POINT_SYSTEM_ID,
+  CONVERSATION_CREATE_EVENT,
+  MESSAGE_SENT_POINT,
+} from "../consts";
+
+const trackNewConversation = async (
```

Comment:
Is it necessary to track the new conversation?
- What problem would emerge if we remove the `trackNewConversation` lib?